pkg/envoy/registry/services_test.go

```diff
@@ -150,6 +201,36 @@ var _ = Describe("Test Proxy-Service mapping", func() {
 
 			pod := tests.NewPodFixture(namespace, "pod-name", tests.BookstoreServiceAccountName, tests.PodLabels)
 			mockKubeController.EXPECT().ListServices().Return(services)
+			mockKubeController.EXPECT().GetEndpoints(gomock.Any()).Return(&v1.Endpoints{
+				ObjectMeta: metav1.ObjectMeta{
+					Namespace: namespace,
+					Name:      service1Name,
+				},
+				Subsets: []v1.EndpointSubset{
+					{
+						Addresses: []v1.EndpointAddress{
+							{
+								IP: "8.8.8.8",
+							},
+						},
+					},
+				},
+			}, nil).Times(1)
+			mockKubeController.EXPECT().GetEndpoints(gomock.Any()).Return(&v1.Endpoints{
```

Comment:
Should this resource be different somehow from the one right above it?

Reply:
Using `.Times(2)` on the single resource then might be a more clear and concise way to express that then.

Reply:
I ended up adding a Headless service to the test, so I needed to split them out again